db/sql/sql.go

```diff
@@ -311,6 +321,10 @@ func (db *DB) Begin() (tx transaction.Transaction, err error) {
 		closed:         false,
 		isolationLevel: transaction.RepeatableRead,
 	}
+	if os.Getenv("FUZZY_DB_TX") == "true" {
```

Comment:
Is this used in gohan unit test? Name is ok and functionality is great, but there is no documentation about how to use and when we should use.